torch/nn/modules/module.py

```diff
@@ -859,12 +859,16 @@ def load(module, prefix=''):
                                self.__class__.__name__, "\n\t".join(error_msgs)))
         return _IncompatibleKeys(missing_keys, unexpected_keys)
 
-    def _named_members(self, get_members_fn, prefix='', recurse=True):
+    def _named_members(self, get_members_fn, get_children_members_fn,
+                       prefix='', recurse=True):
         r"""Helper method for yielding various names + members of modules."""
         memo = set()
         modules = self.named_modules(prefix=prefix) if recurse else [(prefix, self)]
         for module_prefix, module in modules:
-            members = get_members_fn(module)
+            if module is self:
```

Comment:
By doing this here, you're changing the behavior of this functions compared to current master.
Because for all children, you now call `get_children_members_fn` instead of `get_members_fn`. But these two functions are different.